samples/Samples.SqlServer/Samples.SqlServer.csproj

```diff
@@ -1,19 +1,19 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Could we refactor the project name to `Samples.SqlCommand`? What are your thoughts on the name?

Reply:
Renamed everywhere in 5a76e5ca9dab124f8134d8727d9ed8d1aaeafd9c.